storage/boltdb/client.go

```diff
@@ -125,11 +125,14 @@ func (client *Client) GetAll(keys storage.Keys) (storage.Values, error) {
 	err := client.view(func(bucket *bolt.Bucket) error {
 		for _, key := range keys {
 			val := bucket.Get([]byte(key))
-			vals = append(vals, storage.CloneValue(storage.Value(val)))
+			if val == nil {
```

Comment:
It's unclear why this change is necessary here.

Reply:
Also indent the error flow rather than keeping both if / else. See https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
Ok, interesting - good to know, I didn't know that was a preferred style. Thx